cmd/kogito/command/shared/install_operator.go

```diff
@@ -36,15 +36,14 @@ import (
 )
 
 const (
-	defaultOperatorImageName   = "quay.io/kiegroup/kogito-cloud-operator"
-	boxDeployPath              = "../../../../deploy"
-	fileOperatorYaml           = "operator.yaml"
-	fileRoleYaml               = "role.yaml"
-	fileRoleBindingYaml        = "role_binding.yaml"
-	fileServiceAccountYaml     = "service_account.yaml"
-	fileKogitoAppCRDYaml       = "crds/app.kiegroup.org_kogitoapps_crd.yaml"
-	fileKogitoDataIndexCRDYaml = "crds/app.kiegroup.org_kogitodataindices_crd.yaml"
-	docURLInstallOperator      = "https://github.com/kiegroup/kogito-cloud-operator#automatically-in-operatorhub"
+	defaultOperatorImageName = "quay.io/kiegroup/kogito-cloud-operator"
+	boxDeployPath            = "../../../../deploy"
```

Comment:
is there any changes for this path gets changed?

Reply:
We're reading the full path for CRDs instead of each file separately:
 https://github.com/kiegroup/kogito-cloud-operator/pull/112/files#diff-2fe158c45786d9c1198cce237756ffe6R100